packages/muelu/test/scaling/ImportPerformance.cpp

```diff
@@ -168,8 +168,7 @@ Epetra_Map * convert_lightweightmap_to_map(const EpetraExt::LightweightMap & A,
 
 
 Epetra_CrsMatrix* convert_lightweightcrsmatrix_to_crsmatrix(const EpetraExt::LightweightCrsMatrix & A) {
-  RCP<TimeMonitor> tm;
-  tm = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("OptimizedTransfer: Convert: MapConstructor")));
+  TimeMonitor tm(*TimeMonitor::getNewTimer("OptimizedTransfer: Convert: MapConstructor"));
```

Comment:
Thanks for fixing the timers here.   Are you using emacs by any chance?  There are a lot of new tabs in this file.  Please remove them:
```
    <control> x h
    <escape> x untabify
    save and quit
```